corehq/apps/case_search/management/commands/compare_case_properties.py

```diff
@@ -0,0 +1,146 @@
+from django.core.management.base import BaseCommand, CommandError
+from corehq.apps.es.case_search import CaseSearchES
+import timeit
+import json
+
+
+# Returns min, max, average of test suite results list
+def get_stats(results_list):
+    n = len(results_list)
+    if n == 1:
+        return results_list[0], results_list[0], results_list[0]
+    min_value = min(results_list)
+    max_value = max(results_list)
+    avg_value = sum(results_list) / n
+    return min_value, max_value, avg_value
+
+
+def run_baseline_query(query_obj):
+    print("Running baseline query...")
+    result = query_obj.run().total
+    print("Done!")
+    return result
+
+
+def run_test_queries(query_obj, repeats):
+    results = []
+    hits = query_obj.run().total
+    print("Running test queries...")
+    for _ in range(repeats):
+        results.append(timeit.timeit(query_obj.run, number=1))
+    print("Done!")
+    return results, hits
+
+
+def define_baseline_query(domain, case_type=None):
+    if case_type is None:
+        query_obj = (CaseSearchES()
+            .domain(domain))
+    else:
+        query_obj = (CaseSearchES()
+            .domain(domain)
+            .case_type(case_type))
+    return query_obj
```

Comment:
I think we should also limit the size of the returned results so that we don't end up timing all the serialization de-serialization of the data:
```suggestion
    query.size(10)
    return query_obj
```